### ISSUE TITLE
Testnet NFT minting doesn't work 

### ISSUE THREAD

**Issue by SkillerChampion** [hashgraph/hedera-nft-minter#51]
### Description

I am facing this issue for last 5 days at [Testnet Mintbar](https://testnet.mintbar.xyz/)

1 - Blade wallet not connecting at all. When I click on Connect Blade wallet, it automatically gives following error - 
![image](https://github.com/hashgraph/hedera-nft-minter/assets/128652723/dee428d1-e936-4f10-952f-50a97b062c53)

2 - Hashpack connects and goes to final screen to mint collection but NFT preview fails with txn fee error (PS. I have 1000 HBARS in the testnet account)

### Steps to reproduce

For Hashpack error - 
1 - Connect Hashpack
2 - Upload NFT image and fill name, description
3 - Click on next and go to final "Confirm screen"
4 - After confirming, my hashpack wallet gets following txn - 
![image](https://github.com/hashgraph/hedera-nft-minter/assets/128652723/5ed2f9ec-1929-4596-822f-0f5c900b8ab4)

This txn gets approved and immediately following it, I get another NFT mint txn as shown below - 
![image](https://github.com/hashgraph/hedera-nft-minter/assets/128652723/15e7df73-df05-4410-88d3-6325980910d0)

This 2nd txn fails with following error - 
![image](https://github.com/hashgraph/hedera-nft-minter/assets/128652723/23b8050d-0eb2-46f6-992b-911b80dc37ab)

`{ "name": "StatusError", "status": "INSUFFICIENT_TX_FEE", "transactionId": "0.0.3434367@1696326513.179120431", "message": "transaction 0.0.3434367@1696326513.179120431 failed precheck with status INSUFFICIENT_TX_FEE", "contractFunctionResult": null }`

### Hedera network

testnet

### Version

v9.1.0 - Hashpack

### Operating system

Windows

**Comment by Ashe-Oro** [comment]
Hi there, thank you for reporting both of these issues. 

1. Blade wallet incompatibility. 
This is a known issue. The Blade wallet SDK introduced breaking changes recently and we haven't yet updated Mintbar to re-support Blade wallet. Instead, we are waiting to integrate with Hedera Wallet Connect once it's ready. For now, please use Hashpack. 

2. Transaction fee error when using Hashpack v9.10 
Are you able to reproduce this? I was unable to as you can see with tx_id https://hashscan.io/testnet/transaction/1696342693.661191701.  Also, I looked at your testnet account https://hashscan.io/testnet/account/0.0.3434367 and it does seem that you have plenty of testnet hbar to cover the tx. 

I see you are also able to create the NFT Collection, but not mint NFTs into it. Very strange.

**Comment by SkillerChampion** [comment]
> Hi there, thank you for reporting both of these issues.
> 
> 1. Blade wallet incompatibility.
>    This is a known issue. The Blade wallet SDK introduced breaking changes recently and we haven't yet updated Mintbar to re-support Blade wallet. Instead, we are waiting to integrate with Hedera Wallet Connect once it's ready. For now, please use Hashpack.
> 2. Transaction fee error when using Hashpack v9.10
>    Are you able to reproduce this? I was unable to as you can see with tx_id https://hashscan.io/testnet/transaction/1696342693.661191701.  Also, I looked at your testnet account https://hashscan.io/testnet/account/0.0.3434367 and it does seem that you have plenty of testnet hbar to cover the tx.
> 
> I see you are also able to create the NFT Collection, but not mint NFTs into it. Very strange.

Yes, i am able to reproduce it. It's happening everytime. The first NFT collection txn succeeds but not 2nd txn. Actually i have around 9000 hbars in my testnet account as shown below -
![IMG_4009](https://github.com/hashgraph/hedera-nft-minter/assets/128652723/62ce8b16-22cb-4781-ab36-9cf73c0a2e69)

**Comment by Ashe-Oro** [comment]
on failure, can you check the browser's console and post any error messages thrown by Mintbar? 

We've had multiple people try to reproduce this and not able to (yet). It could be a Hashpack issue, but let's check the browser console.

**Comment by SkillerChampion** [comment]
So, I uninstalled and installed hashpack, cleared mintbar site data, then tried and got same error. 
- 
![image](https://github.com/hashgraph/hedera-nft-minter/assets/128652723/179a931b-0aa0-40c9-85cc-3875d233164c)


The logs below comes like 4-5 secs after the txn fails
![image](https://github.com/hashgraph/hedera-nft-minter/assets/128652723/c9457ca8-28f0-4e7f-830f-051c2cec642b)

**Comment by Ashe-Oro** [comment]
ok thank you, we're looking into it.

**Comment by Ashe-Oro** [comment]
hey @SkillerChampion can you try with incognito enabled? The hashpack team also tested this without issues. 

I know you said that you've cleared the mintbar cache, but it looks like an issue identifying your active wallet. Maybe something with the Blade attempt has confused how we check for the active wallet.

**Comment by SkillerChampion** [comment]
> hey @SkillerChampion can you try with incognito enabled? The hashpack team also tested this without issues.
> 
> I know you said that you've cleared the mintbar cache, but it looks like an issue identifying your active wallet. Maybe something with the Blade attempt has confused how we check for the active wallet.

So, I tried in incognito but unable to connect wallet because the icognito mode is missing hashpack extension, but I changed my google account profile on chrome. Added testnet account in hashpack and tried and got same error. 


How in the world is this working for you guys ?? I am getting same error everytime

**Comment by Ashe-Oro** [comment]
`I uninstalled and installed hashpack`

Does this mean that you re-imported the private key into a new hashpack installation?

**Comment by Ashe-Oro** [comment]
we encountered other issues (still haven't been able to reproduce yours) but deleting the account from hashpack and re-importing it via private key seemed to resolve the issues.

**Comment by Ashe-Oro** [comment]
hi again @SkillerChampion ....i spoke with the Hashpack team and they are pushing a new version with a few fixes regarding signing issues. I believe this will resolve your issue, but we need to wait on them.
